neuralforecast/core.py

```diff
@@ -422,6 +451,99 @@ def cross_validation(
         fcsts_df = fcsts_df.merge(df, how="left", on=["unique_id", "ds"])
         return fcsts_df
 
+    def predict_insample(self, step_size: int = 1):
```

Comment:
Can we rename the function inspired by StatsForecast method?
We can also match the documentation with StatsForecast's.

https://nixtla.github.io/statsforecast/core.html#statsforecast.forecast_fitted_values